crates/biome_cli/src/execute/mod.rs

```diff
@@ -304,6 +306,17 @@ impl Execution {
         }
     }
 
+    pub(crate) fn as_write_suppressions_mode(&self) -> bool {
```

Comment:
I think we don't need this function anymore. We need to update `as_fix_file_mode` to return the new variant `FixFileMode::ApplySuppresion`